.github/workflows/distribution.yml

```diff
@@ -9,9 +9,14 @@ jobs:
         os: [ubuntu-latest]
     runs-on: ${{ matrix.os }}
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-java@v1
-      with:
-        java-version: 11
-    - name: Build and publish to Bintray
-      run: ./gradlew :web-utils:publish -DBINTRAY_USER=${{ secrets.BINTRAY_USER }} -DBINTRAY_KEY=${{ secrets.BINTRAY_KEY }} -PlibraryPublish
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Prepare to publish
+        run: |
+          echo '${{secrets.GPG_KEY_CONTENTS}}' | base64 -d > publish_key.gpg
+          gpg --quiet --batch --yes --decrypt --passphrase="${{secrets.SECRET_PASSPHRASE}}" \
+          --output secret.gpg publish_key.gpg
+      - name: Build and publish to mavenCentral
+        run: ./gradlew :web-utils:publish -DOSSRH_USER=${{ secrets.OSSRH_USER }} -DOSSRH_KEY=${{ secrets.OSSRH_KEY }} -PlibraryPublish -Psigning.secretKeyRingFile=secret.gpg -Psigning.keyId=${{secrets.SIGNING_KEYID}} -Psigning.password=${{secrets.SIGNING_PASSWORD}}
```

Comment:
please configure inmemory gpg key.
https://docs.gradle.org/current/userguide/signing_plugin.html#sec:in-memory-keys